gemini/tests/lists.js

```diff
@@ -63,14 +63,14 @@ gemini.suite('lists', (child) => {
             .capture('default');
     });
 
-    gemini.suite('old-lists-in-cards', (child) => {
-        child.setUrl('/lists/old-lists-in-cards')
-            .before((actions, find) => {
-                actions.waitForElementToShow('.card', WAIT_TIME);
-                actions.wait(WAIT_LOAD_TIME);
-            })
-            .setCaptureElements('.row')
-            .capture('default');
-    });
+    // gemini.suite('old-lists-in-cards', (child) => {
```

Comment:
no good reason. thats what we did with the other tests too. we have them for each demo but if that demo is not exported or is old we comment out the test. if you think we should remove them, i can.